src/common/grpc-expr/src/insert.rs

```diff
@@ -216,6 +219,25 @@ fn collect_column_values(column_datatype: ColumnDataType, values: &Values) -> Ve
                 Time::new_nanosecond(*v)
             ))
         }
+        ColumnDataType::IntervalYearMonth => {
+            collect_values!(values.interval_year_month_values, |v| {
+                ValueRef::Interval(Interval::from_year_month(*v))
+            })
+        }
+        ColumnDataType::IntervalDayTime => {
+            collect_values!(values.interval_day_time_values, |v| {
+                ValueRef::Interval(Interval::from_i64(*v))
```

Comment:
`from_day_time()` receives 2 params(`day` and `millisecond`), but v is `&i64` type. If we want to unify function names，I think we should change the `from_month` above to `from_i32`  which may be a better choice. 
And I'm not very good at handling `from_month_day_nano` because it use `IntervalMonthDayNano` rather than native i128 type.